libs/core/langchain_core/messages/ai.py

```diff
@@ -88,6 +96,14 @@ def get_lc_namespace(cls) -> List[str]:
         """Get the namespace of the langchain object."""
         return ["langchain", "schema", "messages"]
 
+    @property
+    def lc_attributes(self) -> Dict:
+        """Attrs to be serialized even if they are derived from other init args."""
+        return {
+            "tool_calls": self.tool_calls,
```

Comment:
why are `tool_call_chunks` serialized properly without being in here?

Reply:
default behavior is to serialize (only) attributes that are passed in as init args